job-templates/kubernetes_containerd.json

```diff
@@ -16,8 +16,7 @@
         },
         "networkPlugin": "azure",
         "containerRuntime": "containerd",
-        "windowsContainerdURL": "https://github.com/marosset/windows-cri-containerd/releases/download/nightly/windows-cri-containerd.zip",
-        "windowsSdnPluginURL": "https://k8swin.blob.core.windows.net/k8s-windows/windows-cni-containerd-custom.zip"
```

Comment:
latest aks-engine removed the check for sdn plugin: https://github.com/Azure/aks-engine/pull/3508